src/WebJobs.Script/Rpc/FunctionRegistration/IFunctionDispatcher.cs

```diff
@@ -7,14 +7,16 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Rpc
 {
-    internal interface IFunctionDispatcher : IDisposable
+    public interface IFunctionDispatcher : IDisposable
     {
-        IDictionary<WorkerConfig, LanguageWorkerState> LanguageWorkerChannelStates { get; }
+        IDictionary<string, LanguageWorkerState> LanguageWorkerChannelStates { get; }
```

Comment:
Should this dictionary be public? Seems like internal state to the dispatcher

Reply:
this list is used by the ScriptHost to populate functionErrors for each function if something goes wrong with the language worker process needed by the function